app/views/group_offers/index.html.slim

```diff
@@ -3,7 +3,7 @@ h1= t_title
 nav.navbar.section-navigation
   ul.list-inline
     li= button_link t_title(:new), new_group_offer_path, 'success', dimension: 'sm'
-    - if policy(GroupOfferCategory).superadmin?
+    - if policy(GroupOfferCategory).superadmin_or_department_manager?
```

Comment:
I think that is now solved from the GroupOffer policies

Reply:
Not sure, but in any case you should check for the action (`index?`) here rather than the role, otherwise you're duplicating the logic again.

In other words: you're rendering a link to the `index` action with `group_offer_categories_path`, so you should check if the user has permission for this action (`policy(GroupOfferCategory).index?`), instead of repeating the check for the user's role (which is already in `GroupOfferCategoryPolicy#index?`).